source/includes/fundamentals/code-snippets/Cursor.java

```diff
@@ -44,10 +44,20 @@ private void manualIteration(){
             System.out.println(cursor.next().toJson());
         }
         // end manualIteration
+    }
 
-        // begin close 
-        cursor.close();
-        // end close 
+    private void closeExample(){
+        // begin closeExample
+        MongoCursor<Document> cursor = collection.find().cursor();
+        
+        try{
```

Comment:
Do you want to show try-with-resources (introduced in Java 7)

    try (MongoCursor<Document> cursor = collection.find().cursor()) {
         while (cursor.hasNext()){
            System.out.println(cursor.next().toJson());
    }

Reply:
hm, doing this would mean the finally block wouldn't work to close the cursor.  Would I in this case put the close after the while loop in the try block?

Reply:
If you do that you don't need the finally block, or the call to close.  See https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html